app/domain/export/pdf/messages/letter_with_invoice/donation_confirmation.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Die Mitte Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Export::Pdf::Messages::LetterWithInvoice
+  class DonationConfirmation < Export::Pdf::Messages::Letter::Section
+    def initialize(pdf, letter, recipient, options)
+      super(pdf, letter, options)
+      @recipient = recipient
+    end
+
+    def render
+      donation_text = stamped(:donation_confirmation_text) { donation_confirmation_text }
+      text([donation_text, last_year_donation_amount])
+    end
+
+    private
+
+    def donation_confirmation_text
+      "\n " + I18n.t('messages.export.section.donation_confirmation')
+    end
+
+    def last_year_donation_amount
+      currency = letter.invoice.currency
+
+      donation_amount = Donation.new.
+                                in_last(1.year).
+                                in_layer(letter.group).
+                                of_person(@recipient).
+                                previous_amount.
+                                to_s
```

Comment:
Jetzt bin ich nur noch am "Tüpflischiisse", aber ich würde einen Space mehr vor diesen Zeilen machen, sodass sie mit `new.` aligned sind

Reply:
Ich würde noch weiter "Tüpflischiisse", nimmt man das `.` nicht direkt auf die newline?
Also zB:
```

Reply:
Das ist sehr Geschmackssache, siehe https://github.com/rubocop/ruby-style-guide#multi-line-method-chains und die riesenlange Diskussion auf https://github.com/rubocop/ruby-style-guide/pull/176
Der Punkt am Ende der Zeile ist verträglicher wenn man Code in die Rails Konsole rüberkopiert. Aber davon abgesehen kenne ich kaum ausschlaggebende Argumente für eine der Optionen.